CMakeLists.txt

```diff
@@ -13,9 +13,20 @@ project(git
 		VERSION ${git_version}
 		LANGUAGES C)
 
+#[[
+NOTE:
+FOR LINUX USE THIS GIT REPO https://github.com/git/git
+FOR WINDOWS USE THIS GIT REPO https://github.com/git-for-windows/git
+]]
+
 #TODO gitk git-gui gitweb
+#TODO Enable NLS on windows natively
 #TODO Add pcre support
 
+#NOTE
+#On windows when building with shared libraries
```

Comment:
"windows" are things that are put into walls so that sunlight can come in. If you are referring to the operating system, the first "W" needs to be upper-case ;-)